### PR TITLE
Redis documentation

### PR DESCRIPTION
**Pull request by BharatKalluri**

### CODE REVIEW

**Comment by pvalena** [comment]
Hello! Thanks for your PR.

File: tech/database/redis/about.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+$ sudo systemctl enable redis
+```
+
+To test redis-cli, run
+
+```console
+$ redis-cli ping
+```
+
+the result from the above command will be
```
Comment (pvalena) [inline tech/database/redis/about.md:33]: Please let's merge this into one block, with the above. Command output should be obvious (this is consistent with the rest of the website AFAIK).

File: tech/database/redis/about.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Configuring Redis
+
+The config file for Redis is located at `/etc/redis/redis.conf`.
+
+```console
+$ sudo nano /etc/redis/redis.conf
+```
+
+Redis should be always restarted after changing settings. Restart redis by running
```
Comment (pvalena) [inline tech/database/redis/about.md:47]: I'm just curious, isn't there `reload`?

File: tech/database/redis/about.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+```console
+$ sudo systemctl restart redis
+```
+
+Secure redis by enabling authentication. Add this line in `/etc/redis/redis.conf`
+
+```console
+$ requirepass  <AuthPassword>
```
Comment (pvalena) [inline tech/database/redis/about.md:56]: `$ ` denotates command to be run. Please remove it.

**Review by pvalena — CHANGES_REQUESTED** [review]
Looks good, let's just polish the formatting.

For reference / more info:
https://developer-portal.github.io/contributing/write-general-writing-hints.html

**Comment by pvalena** [comment]
Please also create corresponding PR (if there's not one already) with the logo in `website`  repository:

https://github.com/developer-portal/website/tree/master/static/logo

File: tech/database/redis/about.md (hunk 2 above)
Comment (BharatKalluri) [reply tech/database/redis/about.md:42]: Restart is the suggested approach for config changes if the config file on disk is changed

**Comment by BharatKalluri** [comment]
Thanks for the review @pvalena !

> Please also create corresponding PR (if there's not one already) with the logo in `website` repository:
> 
> https://github.com/developer-portal/website/tree/master/static/logo

I added redis logo previously at https://github.com/developer-portal/website/blob/master/static/logo/redis.png

File: tech/database/redis/about.md [hunk 4]
```diff
@@ -53,7 +48,7 @@ $ sudo systemctl restart redis
 Secure redis by enabling authentication. Add this line in `/etc/redis/redis.conf`
 
 ```console
-$ requirepass  <AuthPassword>
```
Comment (pvalena) [inline tech/database/redis/about.md]: Just to be sure: do I understand correctly that this is the line user should add to the file above?

**Comment by pvalena** [comment]
Thanks! LGTM.

**Pull request merged**